kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml

```diff
@@ -69,13 +69,14 @@
    </p>
             <div xml:id="kzz7yh-f47354-Dd1e122">
                <head xml:id="kzz7yh-f47354-Hd1e124">Quaestio 1</head>
+               <head type="question-title">Utrum deus die septimo non requieuerit<!--TODO word?-->> </head>
```

Comment:
```suggestion
               <head type="question-title">Utrum deus die septimo non requieuerit</head>
```

Reply:
"requiesco, -ere, requievi, requietum" - 3rd. sg. perfect active subjunctive form; "to rest"